routes/sessionRoutes.js

```diff
@@ -1,6 +1,8 @@
 const express = require("express");
 const passport = require("passport");
+const csrf = require('csurf');
```

Comment:
Requires change:
Swap this for the `csrfProtection` middleware mentioned in a previous review comment above.